src/services/MeasureService.ts

```diff
@@ -117,4 +125,36 @@ export class MeasureService implements Service<fhir4.Measure> {
     logger.info('Successfully generated $data-requirements report');
     return results;
   }
+
+  /**
+   * result of sending a POST request to:
+   * {BASE_URL}/4_0_1/Measure/$submit or {BASE_URL}/4_0_1/Measure/:id/$submit
+   * POSTs a new artifact in "draft" status. The operation results in an error if the artifact
+   * does not have status set to "draft."
+   */
+  async submit(args: RequestArgs, { req }: RequestCtx) {
+    logger.info(`${req.method} ${req.path}`);
+
+    const contentType: string | undefined = req.headers['content-type'];
+    checkContentTypeHeader(contentType);
+
+    const resource = req.body;
+    checkExpectedResourceType(resource.resourceType, 'Measure');
+
+    // check for "draft" status on the resource
+    if (resource.status !== 'draft') {
+      throw new BadRequestError(`The artifact must be in 'draft' status.`);
+    }
+
+    // lastUpdated should be second because it should overwrite a meta.lastUpdated tag in the request body
```

Comment:
I am a little confused about this comment...looks like it was leftover from the implementation of create in the test-server. Maybe clarify or just take it out?